core/templates/components/rubrics-editor/rubrics-editor.component.ts

```diff
@@ -95,6 +97,33 @@ export class RubricsEditorComponent {
     return Boolean(this.editableExplanations[difficulty][index]);
   }
 
+  isExplanationLengthValid(difficulty: string, idx: number): boolean {
+    return this.editableExplanations[difficulty][idx].length <=
+    this.maximumCharacterLengthOfExplanation;
+  }
+
+  isMediumLevelExplanationValid(): boolean {
+    // Checking if medium level rubrics have atleast one explantion.
+    this.selectedRubricIndex = 1;
+    if (this.rubrics[this.selectedRubricIndex] === undefined) {
+      return false;
+    }
+    return (this.rubrics[this.selectedRubricIndex].getExplanations().length >=
+    1);
```

Comment:
```suggestion
    return (
        this.rubrics[this.selectedRubricIndex] &&
        this.rubrics[this.selectedRubricIndex].getExplanations().length >= 1);
```

Reply:
I guess it will raise an exception for undefined, I'm not sure though.
Thanks

Reply:
This is called a "short-circuit evaluation" and is a commonly used Javascript statement (see [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND#short-circuit_evaluation)).

Reply:
Got it, thanks.
I'll update it.